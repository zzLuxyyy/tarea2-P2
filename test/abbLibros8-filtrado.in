crearABBLibrosVacio

filtradoPorGeneroABBLibros 1

crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
insertarLibroABBLibros

filtradoPorGeneroABBLibros 3
filtradoPorGeneroABBLibros 1
filtradoPorGeneroABBLibros 2
filtradoPorGeneroABBLibros 4

crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
insertarLibroABBLibros

crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros

crearFecha 01/01/1869
crearLibro 101112131 "Guerra y paz" "León" "Tolstói" "Épica historia sobre la nobleza rusa durante las guerras napoleónicas" 3
insertarLibroABBLibros

crearFecha 05/04/1974
crearLibro 311122334 "Carrie" "Stephen" "King" "Una adolescente con poderes telequinéticos enfrenta el abuso y el horror" 2
insertarLibroABBLibros

crearFecha 19/10/1953
crearLibro 151617181 "Fahrenheit 451" "Ray" "Bradbury" "Un bombero quema libros en una sociedad donde leer es delito" 2
insertarLibroABBLibros

crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
insertarLibroABBLibros

crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros

crearFecha 17/09/1951
crearLibro 194512348 "Bestiario" "Julio" "Cortázar" "Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano" 1
insertarLibroABBLibros

crearFecha 01/01/1952
crearLibro 121314151 "El viejo y el mar" "Ernest" "Hemingway" "Un viejo pescador enfrenta al mar en una lucha silenciosa" 1
insertarLibroABBLibros

crearFecha 01/01/1989
crearLibro 224512789 "El palacio de la luna" "Paul" "Auster" "Un joven cae en la pobreza y encuentra sentido en un viaje inesperado" 1
insertarLibroABBLibros

crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
insertarLibroABBLibros

# 1 elemento
filtradoPorGeneroABBLibros 7

# 2 elementos
filtradoPorGeneroABBLibros 3

# 4 elementos
filtradoPorGeneroABBLibros 2

# 5 elementos
filtradoPorGeneroABBLibros 1

liberarABBLibros
Fin
