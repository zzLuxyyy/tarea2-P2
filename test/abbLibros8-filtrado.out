1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>3>4>5>Se insertó el libro con ISBN 140003446 al árbol.
6>Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
7>8>9>10>11>12>Se insertó el libro con ISBN 204578912 al árbol.
13>14>15>Se insertó el libro con ISBN 316769177 al árbol.
16>17>18>Se insertó el libro con ISBN 101112131 al árbol.
19>20>21>Se insertó el libro con ISBN 311122334 al árbol.
22>23>24>Se insertó el libro con ISBN 151617181 al árbol.
25>26>27>Se insertó el libro con ISBN 999001122 al árbol.
28>29>30>Se insertó el libro con ISBN 843397030 al árbol.
31>32>33>Se insertó el libro con ISBN 194512348 al árbol.
34>35>36>Se insertó el libro con ISBN 121314151 al árbol.
37>38>39>Se insertó el libro con ISBN 224512789 al árbol.
40>41>42>Se insertó el libro con ISBN 880619315 al árbol.
43>#  1 elemento.
44>Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
45>#  2 elementos.
46>Libro 101112131
Título: Guerra y paz
Autor: León Tolstói
Descripción: Épica historia sobre la nobleza rusa durante las guerras napoleónicas
Género: 3
Fecha de edición: 1/1/1869
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
47>#  4 elementos.
48>Libro 151617181
Título: Fahrenheit 451
Autor: Ray Bradbury
Descripción: Un bombero quema libros en una sociedad donde leer es delito
Género: 2
Fecha de edición: 19/10/1953
Libro 311122334
Título: Carrie
Autor: Stephen King
Descripción: Una adolescente con poderes telequinéticos enfrenta el abuso y el horror
Género: 2
Fecha de edición: 5/4/1974
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
49>#  5 elementos.
50>Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
Libro 194512348
Título: Bestiario
Autor: Julio Cortázar
Descripción: Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano
Género: 1
Fecha de edición: 17/9/1951
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
51>52>Fin.
